### ISSUE TITLE
Http Post with Content-Type: application/json;v=3

### ISSUE THREAD

**Issue by arun0009** [vlucas/frisby#288]
How can i set Content-Type as application/json;v=3 with Frisbyjs? If I set the Content-Type and don't set json: true body isn't showing as json entity on my api server. 

If I set Content-Type: application/json;v=3 and set json:true it overrides Content-Type to be application/json.

<code>
In the Extended BNF notation of RFC 822, a Content-Type header field value is defined as follows:
Content-Type := type "/" subtype *[";" parameter]
</code>

**Comment by sasikanthnimmagadda** [comment]
When header 'content-type' is specified as 3rd argument to post(url, body, options) 
then the release implementation (as taken from npm) will over ride and default it to 'application/json'

This has been fixed by #266 , #259 

You may follow  this workaround
```
var frisby=require('frisby');
frisby.create('test')
.post('http://httpbin.org/post',
    {
        "input": "data",
        "url": "http://httpbin.org/get"
    },
    {
        json:true
    }
)
.addHeader('content-type', 'user-specific') //this will work.
.inspectRequest()
.toss();
```

The below will not work for you, unless you pick the latest from master branch or wait for a release.
```
var frisby=require('frisby');
frisby.create('test')
.post('http://httpbin.org/post',
    {
        "input": "data",
        "url": "http://httpbin.org/get"
    },
    {
        json:true,
        headers : {'content-type':'user-specific'}
       //when specified here, current implementation will over ride
       // and default it to 'application/json'
    }
)
.inspectRequest()
.toss();
```

~ Sasi

**Comment by arun0009** [comment]
Thanks @sasikanthnimmagadda it worked like a charm. 

Appreciate your detailed response.
